requirements.txt

```diff
@@ -1,3 +1,4 @@
 django-freeradius<0.2.0
 openwisp-users>=0.1.12,<0.2
 django-rest-auth>=0.9.3,<0.12.0
+openwisp-utils[qa]>=0.4.1
```

Comment:
here we don't need it because it's defined in django-freeradius, I already updated it there.